### ISSUE TITLE
[Druid] Dash Can Be Used Outside of Cat Form

### ISSUE THREAD

**Issue by arestavo** [dalaranwow/dalaran-wow#5731]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
One can use dash out of cat form, in tree form, in bear form, in travel form, and I assume in moonkin form. even in regular caster form.
**Expected behaviour**: Tell us what should happen instead
Dash should be grayed out and not be activatable unless in cat form.
**Steps to reproduce the problem**:

1. Press Dash outside of cat form.
2. It activates.
3. It shouldn't let you activate the skill
![WoWScrnShot_052819_171709](https://user-images.githubusercontent.com/50332138/58518448-7de32500-816c-11e9-87c5-d7f3e6b741b3.jpg)
 unless you are in Cat form.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Chenocide** [comment]
It's usable but it has no effect.

**Comment by arestavo** [comment]
Yes, and it should NOT be usable when you aren't in cat form. Hence the ticket.

**Comment by arestavo** [comment]
> It's usable but it has no effect.

Yes, and it should NOT be usable when you aren't in cat form. Hence the ticket.
